guide/C/ch_invest.xml

```diff
@@ -140,27 +140,26 @@
         </varlistentry>
 
         <varlistentry>
-          <term><emphasis>Realized vs Unrealized Gain/Loss</emphasis></term>
+          <term>Realized vs Unrealized Gain/Loss</term>
 
           <listitem>
             <para>Unrealized gain or loss occurs when you&rsquo;ve got a change in price of an asset. You realize the
-              gain/loss when you actually sell the asset. See also <emphasis>capital
-              gain/loss</emphasis>.
+              gain/loss when you actually sell the asset. See also <xref linkend="invest_terms2.capgain" />.
```

Comment:
Note I had written this in the assumption you were using a glosslist.

xref seems not to work very well in yelp. it displays
> See also Terminology.

and jumps to the begin of the page.
xref is searching for a title, but term has none.
In HTML it has no hyperlink, but perhaps the stylesheet parameter glossterm.auto.link to 1 is missing.
From http://www.sagehill.net/docbookxsl/LinkToGlossary.html shouldn't it be
`<glossterm linkend="invest_terms2.capgain">Capital gains</glossterm>`

Perhaps `<glossseealso otherterm="invest_terms2.capgain">Capital gains</glossseealso>` would be the better tag here?